### PR TITLE
DKP-285: Add release notes for 4.11.1

### PR DESCRIPTION
**Pull request by gavin-timbrell**

### CODE REVIEW

**Comment by aevesdocker** [comment]
Thanks for the PR @gavin-timbrell. All looks good, but the Linux install page also needs updating with the latest builds https://deploy-preview-15316--docsdocker.netlify.app/desktop/install/linux-install/

**Comment by gavin-timbrell** [comment]
> Thanks for the PR @gavin-timbrell. All looks good, but the Linux install page also needs updating with the latest builds https://deploy-preview-15316--docsdocker.netlify.app/desktop/install/linux-install/

Done, not sure how I forgot about that 😅

**Review by aevesdocker — APPROVED** [review]

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,12 +24,53 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed `docker login` to private registries when Registry Access Management is enabled and access to Docker Hub is blocked [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
+
```
Comment (stephen-turner) [inline desktop/release-notes.md:39]: There are two critical bug fixes in this release, why is only one listed?

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,12 +24,53 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed `docker login` to private registries when Registry Access Management is enabled and access to Docker Hub is blocked [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
```
Comment (stephen-turner) [inline desktop/release-notes.md:38]: Lots of people on the original ticket claimed that they had the issue even when RAM wasn't enabled. In fact, several of them weren't even on Business subscriptions. So (1) are we sure it's fixed in that case? (2) if so, the release note doesn't accurately describe the impact.

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: desktop/release-notes.md (hunk 1 above)
Comment (p1-0tr) [reply desktop/release-notes.md:39]: Sorry, my bad, I must have missed the release note field in my PR.

`- Fixed regression preventing VM system locations (e.g. /var/lib/docker) from being bind mounted [for-mac/issues#6433](https://github.com/docker/for-mac/issues/6433)`

File: desktop/release-notes.md (hunk 2 above)
Comment (fredericdalleau) [inline desktop/release-notes.md:38]:
```suggestion
- Fixed `docker login` to private registries from WSL2 distro [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
```

Comment (gavin-timbrell) [reply desktop/release-notes.md:38]: Thanks for that, does this mean it's a windows-only issue @fredericdalleau?

Comment (fredericdalleau) [reply desktop/release-notes.md:38]: I don't know tbh, I've gone only through one diagnostic, and reproduce only on wsl2 distro (and it's a for-win) so it's a reasonable bet.

**Review by fredericdalleau — APPROVED** [review]
LGTM

**Review by stephen-turner — APPROVED** [review]
LGTM now, thanks.

**Pull request merged**